Python/bootstrap_frozen.c

```diff
@@ -0,0 +1,45 @@
+
+/* Frozen modules bootstrap */
```

Comment:
This file is essentially identical to Python/frozen.c if we cut out the non-bootstrap parts, right?  So could we just use frozen.c with something like `#ifndef FROZEN_BOOTRAP` around the non-bootstrap parts?

Otherwise I recommend generating the parts of this file with Tools/scripts/freeze_modules.py, the same as with frozen.c.  If we don't then this will break when someone adds another module to the `BOOTSTRAP` list in freeze_modules.py.